File: Search/app/src/main/java/com/example/rough/apiclient.java
package com.example.rough;

import retrofit2.Call;
import retrofit2.http.GET;
import retrofit2.http.Query;

public interface apiclient {
    //https://api.themoviedb.org/3/search/movie?api_key=2147294b386ecadcb05cd19bcbbdddb9&query=mimi
    @GET("3/search/movie")
    Call<ResponseModel> getSearchedMovie(@Query("apiKey") String apikey, @Query("movieName") String movieName);
}

File: 2.Kotilin_Api_CallingInFragment/app/src/main/java/com/example/fetchdata/Interface/ApiClient.kt
package com.example.fetchdata.Interface

import com.example.fetchdata.Model.ResponseDTO
import retrofit2.Call
import retrofit2.http.GET
import retrofit2.http.Path

interface ApiClient {

    @GET("/api/users/{Id}")
    fun getUserDetails(@Path("Id") id: Int): Call<ResponseDTO>
}